substrate/frame/support/test/tests/runtime_metadata.rs

```diff
@@ -67,13 +67,14 @@ sp_api::decl_runtime_apis! {
 	///
 	/// Documentation on multiline.
 	#[deprecated]
+	#[allow(deprecated)]
```

Comment:
Are all these `deprecated` & `allow(deprecated)` together just done for testing? Does standard rust accepts this as well?